src/decode.c

```diff
@@ -2637,7 +2637,7 @@ static void setup_tile(Dav1dTileState *const ts,
                        const Dav1dFrameContext *const f,
                        const uint8_t *const data, const size_t sz,
                        const int tile_row, const int tile_col,
-                       const int tile_start_off)
+                       const unsigned tile_start_off)
```

Comment:
We don't need to backport this because we already changed `tile_start_off` to a `usize`, right?

Reply:
You are correct.